{
  "uid" : "351c8700c3401171",
  "name" : "User tries to pay a cart with two RPTs that exists already in GPD in invalid state",
  "fullName" : "User pays a payment carts from existing payment position via nodoInviaCarrelloRPT: User tries to pay a cart with two RPTs that exists already in GPD in invalid state",
  "historyId" : "3d26ee416cc3affe0103c3d8164ab3c5",
  "time" : {
    "start" : 1745781339146,
    "stop" : 1745781344615,
    "duration" : 5469
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
  "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.3/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.3/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.3/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.3/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
    "steps" : [ {
      "name" : "Given systems up",
      "time" : {
        "start" : 1745781339146,
        "stop" : 1745781339891,
        "duration" : 745
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Given a cart of RPTs non-multibeneficiary",
      "time" : {
        "start" : 1745781339891,
        "stop" : 1745781339891,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And a single RPT of type BBT with 1 transfers of which none are stamps",
      "time" : {
        "start" : 1745781339891,
        "stop" : 1745781339892,
        "duration" : 1
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And a single RPT of type BBT with 1 transfers of which none are stamps",
      "time" : {
        "start" : 1745781339892,
        "stop" : 1745781339892,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And an existing payment position related to first RPT with segregation code equals to 48 and state equals to DRAFT",
      "time" : {
        "start" : 1745781339892,
        "stop" : 1745781340283,
        "duration" : 391
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "3b459237b99a57eb",
        "name" : "Step: [an existing payment position related to first RPT with segregation code equals to 48 and state equals to DRAFT] - Sent request",
        "source" : "3b459237b99a57eb.txt",
        "type" : "text/plain",
        "size" : 1260
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "When the user tries to pay a cart of RPTs on EC website",
      "time" : {
        "start" : 1745781340284,
        "stop" : 1745781340829,
        "duration" : 545
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "5b61e410aacc739a",
        "name" : "Step: [the user tries to pay a cart of RPTs on EC website] - Sent request",
        "source" : "5b61e410aacc739a.txt",
        "type" : "text/plain",
        "size" : 15119
      }, {
        "uid" : "96673422e51fe159",
        "name" : "Step: [the user tries to pay a cart of RPTs on EC website] - Received response",
        "source" : "96673422e51fe159.txt",
        "type" : "text/plain",
        "size" : 409
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "Then conversion to new model fails in wisp-converter",
      "time" : {
        "start" : 1745781340829,
        "stop" : 1745781342059,
        "duration" : 1230
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "ae4771926a35c70e",
        "name" : "Step: [conversion to new model fails in wisp-converter]: on sessionId: [70e84cd3-58e8-434e-8278-b8fefbeb4770] - Received response",
        "source" : "ae4771926a35c70e.txt",
        "type" : "text/plain",
        "size" : 5807
      }, {
        "uid" : "6b412a8457e3d36f",
        "name" : "Received response",
        "source" : "6b412a8457e3d36f.attach",
        "type" : "text/html",
        "size" : 5677
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "And the KO receipt is sent",
      "time" : {
        "start" : 1745781342059,
        "stop" : 1745781344614,
        "duration" : 2555
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.3/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.3/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "7e23fd94b28885e9",
        "name" : "Step: [the KO receipt is sent]: on IUV: [090359499712888] - Received response",
        "source" : "7e23fd94b28885e9.txt",
        "type" : "text/plain",
        "size" : 54921
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 8,
    "attachmentsCount" : 6,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "tag",
    "value" : "nodo_invia_carrello_rpt"
  }, {
    "name" : "tag",
    "value" : "runnable"
  }, {
    "name" : "tag",
    "value" : "unhappy_path"
  }, {
    "name" : "feature",
    "value" : "User pays a payment carts from existing payment position via nodoInviaCarrelloRPT"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "runnable", "nodo_invia_carrello_rpt", "unhappy_path" ]
  },
  "source" : "351c8700c3401171.json",
  "parameterValues" : [ ]
}